stl/inc/iomanip

```diff
@@ -57,8 +57,6 @@ struct _Monobj { // store reference to monetary amount
 
     _Money& _Val; // the monetary amount reference
     bool _Intl; // international flag
-
-    _Monobj& operator=(const _Monobj&) = delete;

```

Comment:
No change requested: I vaguely recall that these were deleted because of obnoxious compiler warnings of the form "whoa, there's a reference data member, I won't be able to generate an assignment operator, and I'm going to warn about this even before you attempt to assign any objects". I believe that these warnings were fixed :tada: and if we're not seeing them in the test suite then we're good to go.